src/request/request.cpp

```diff
@@ -199,38 +182,37 @@ bool Request::parseRequestHeaders(std::istringstream &requestStream)
 std::string Request::parseRequestBody(const std::string &_rawRequest)
 {
 	size_t body_start;
+	std::string body;
 
 	body_start = _rawRequest.find(CRLFCRLF);
-	if (body_start == std::string::npos) {
+	if (body_start == std::string::npos)
 		return "";
-	}
-	std::string body = _rawRequest.substr(body_start + 4, _contentLength);
+	body = _rawRequest.substr(body_start + 4, _contentLength);
```

Comment:
kunnen we de 4 aanpassen? of een check toevoegen? mogelijk naar iets eleganters en meer foolprove. weet dat ik ook naar zat te kijken en heb het toen niet aangepast.

Reply:
Ik check hierboven al of CRLFCRLF the vinden is en dat zijn 4 chars. Dus als dat het geval is moet "body_start + 4" ook geen probleem zijn.